### PR TITLE
Optimize:optimize load balancer code and add testcase

### PR DESCRIPTION
**Pull request by lingxiao-wu**
## PR Type

Code style update (formatting, local variables).

## Checklist

- [x] Add copyright holder at the beginning of .class file if it is new.
- [x] Add information of this PR to CHANGELOG.md in root of project.
- [x] All junit tests passing.
- [ ] Coverage from `Codecov Report` should not decrease.

## Checklist (Optional)

- [ ] Will Pull Request to branch of 2020.0 and 2021.0.
- [x] Add documentation in javadoc in code or comment below the PR if necessary.
- [x] Add your name as @author to the beginning of .class file.

### CODE REVIEW

**Review by SkyeBeFreeman — CHANGES_REQUESTED** [review]
Add info of this PR to CHANGELOG.md in the root of project.

**Comment by SkyeBeFreeman** [comment]
@lingxiao-wu resolve the conflicts.

File: spring-cloud-tencent-polaris-loadbalancer/src/main/java/com/tencent/cloud/polaris/loadbalancer/LoadBalancerUtils.java [hunk 1]
```diff
@@ -37,37 +35,50 @@
...
+	private static final int DEFAULT_WEIGHT = 100;
 
+	/**
+	 * transfer servers to ServiceInstances.
+	 *
+	 * @param servers servers
+	 * @return ServiceInstances
+	 */
+	public static ServiceInstances transferServersToServiceInstances(Flux<List<ServiceInstance>> servers) {
+		List<Instance> instances = servers.toStream()
```
[the remark this answers is not in the document]
Comment (lingxiao-wu) [reply spring-cloud-tencent-polaris-loadbalancer/src/main/java/com/tencent/cloud/polaris/loadbalancer/LoadBalancerUtils.java]: 我已经改回去了

**Review by SkyeBeFreeman — APPROVED** [review]
LGTM.

**Pull request merged**